source/includes/documentation/_vue_js_elements.md

```diff
@@ -1892,25 +1892,87 @@ For examples of usage see: [JDI vuetify page tests for sparklines](https://githu
+@UI("#InsetSubheader .v-subheader")
+public static Subheader insetSubheader;
+
+@Test(description = "Test checks inset Subheader features: 'theme' and 'text'")
+public void insetSubheaderTest() {
+  insetSubheader.is().displayed();
+  insetSubheader.is().lightTheme();
+  insetSubheader.is().inset();
+  insetSubheader.is().text("Subheader");
 ```
 
-The Subheader component is used to separate sections of lists.
+[Vuetify documentation page](https://vuetifyjs.com/en/components/subheaders/)
+
+__Java__: _com.epam.jdi.light.vuetify.elements.common.Subheader.java_
+
+__Subheader__ - The Subheader component is used to separate sections of lists.
 
 ![Subheaders example](../../images/vuetify/subheader.png)
 
-For examples of usage see: [JDI vuetify page tests for subheaders](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SubheaderTests.java)
+__Vuetify v2.6.14__ code example:
+
+```html
+<div class="v-card v-sheet theme--light">
+  <div class="v-subheader v-subheader--inset theme--light"> Subheader </div>
+  <div role="list" class="v-list v-sheet theme--light">
+    <div tabindex="-1" role="listitem" class="v-list-item theme--light">
+      <div class="v-list-item__action">
+        <i aria-hidden="true" class="v-icon notranslate mdi mdi-label theme--light"></i>
+      </div>
+      <div class="v-list-item__content">
+        <div class="v-list-item__title">List item 1</div>
+      </div>
+    </div>
+    <hr role="separator" aria-orientation="horizontal" class="v-divider v-divider--inset theme--light">
+    <div tabindex="-1" role="listitem" class="v-list-item theme--light">
+      <div class="v-list-item__action">
+        <i aria-hidden="true" class="v-icon notranslate mdi mdi-label theme--light"></i>
+      </div>
+      <div class="v-list-item__content">
+        <div class="v-list-item__title">List item 2</div>
+      </div>
+    </div>
+  </div>
+</div>
+```
+
+| Method                                      | Description                                                                | Return Type     |
+|---------------------------------------------|----------------------------------------------------------------------------|-----------------|
+| **is()**                                    | Returns Assert class                                                       | SubheaderAssert |
+| **isInset()**                               | Checks if element has "v-subheader--inset" class                           | boolean         |
+| **has()**                                   | Returns Assert class                                                       | SubheaderAssert |
+| **classes()**                               | Gets all element's classes as list                                         | List<String>    |
+| **doubleClick()**                           | Double clicks on the element                                               | void            |
```

Comment:
why are some lines red? is it ok?

Reply:
Everything is ok in IDE editor and in preview